dist/pathfora.js

```diff
@@ -143,6 +143,7 @@
   // FUTURE Move to separate files and concat
   /* eslint-disable indent */
   var templates = {
+  'templates': {},
```

Comment:
cool, not a deal breaker. we can always open another ticket to figure it out. not going to hurt n e thing.